ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -324,17 +325,17 @@ void fetchAndRunCommands() {
     }
   }
 
+  @SuppressFBWarnings("DMI_INVOKING_TOSTRING_ON_ARRAY")
   private void executeStatement(final QueuedCommand queuedCommand) {
-    final String commandStatement =
-        queuedCommand.getAndDeserializeCommand(commandDeserializer).getStatement();
-    LOG.info("Executing statement: " + commandStatement);
+    final String commandId = queuedCommand.getCommandId().toString();
```

Comment:
This method is marked as visible for testing: https://github.com/confluentinc/ksql/blob/9c74ff7cf9880317a55690ca291995b7315179c2/ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/QueuedCommand.java#L59

Looks like you're meant to use `queuedCommand. getAndDeserializeCommandId()` instead, and then if you `toString()` you should be able to remove the `DMI_INVOKING_TOSTRING_ON_ARRAY` suppression as well.